gradle/dependencies.gradle

```diff
@@ -45,6 +45,8 @@ versions += [
   snappy: "1.1.2.4",
   zkclient: "0.8",
   zookeeper: "3.4.6",
+  curator: "2.9.0",
```

Comment:
Do we really need this? It would be better to avoid adding third-party dependencies for tests unless we really need them.